mopidy_tidal/library.py

```diff
@@ -168,47 +165,52 @@ def get_images(self, uris):
                     parts = uri.split(':')
                     if parts[1] == 'artist':
                         artist_id = parts[2]
-                        img_uri = self.lru_artist_img.hit(artist_id)
+                        img_uri = self.lru_artist_img.get(artist_id)
                         if img_uri is None:
                             artist = session.get_artist(artist_id)
-                            img_uri = artist.image
-                            self.lru_artist_img[artist.id] = img_uri
+                            img_uri = self._get_image_uri(artist)
+                            if img_uri:
+                                self.lru_artist_img[artist.id] = img_uri
 
                         uri_images = [Image(uri=img_uri, width=512, height=512)]
                     elif parts[1] == 'album':
                         album_id = parts[2]
-                        img_uri = self.lru_album_img.hit(album_id)
+                        img_uri = self.lru_album_img.get(uri)
```

Comment:
Shouldn't it be get(album_id) instead?

Reply:
@BlackLight I do not think this works as intended. I have added some additional log messages to verify. Cache miss always occurs, even after first initialization. Try to add the following log output and you'll see what I mean:

```
...
elif parts[1] == 'album':
album_id = parts[2]
img_uri = self.lru_album_img.get(uri)

if img_uri is None:
>>    logger.info("adding %s to cache %s", parts[1], parts[2])
    album = session.get_album(album_id)
    img_uri = self._get_image_uri(album)
    if img_uri:
        self.lru_album_img[album_id] = img_uri
...
```